modules/apps/object/object-rest-impl/rest-openapi.yaml

```diff
@@ -476,6 +476,34 @@ paths:
                             schema:
                                 $ref: "#/components/schemas/ObjectEntry"
             tags: ["ObjectEntry"]
+    "/{objectEntryId}/object-actions/{objectActionName}":
+        put:
+            parameters:
+                - in: path
+                  name: objectEntryId
+                  required: true
+                  schema:
+                      format: int64
+                      type: integer
+                - in: path
+                  name: objectActionName
+                  required: true
+                  schema:
+                      type: string
+            requestBody:
+                content:
+                    application/json:
+                        schema:
+                            type: string
```

Comment:
This is a string but in the code you're using it to parse a JSON object.

1. Can you declare a schema for that object? Which properties will it have? 
2. If you don't know the answer for 1, at least it should be a free form object `type: object` instead of a string.

Reply:
Hi @4lejandrito,

Thanks for your review!

I talked to Gabriel and Marcela and it was decided that, for now, we won't need the `requestBody`.

I just updated the PR removing the `requestBody`.

Thanks!